src/content/docs/identityserver/quickstarts/2-interactive.md

```diff
@@ -542,82 +542,90 @@ Adding support for external authentication to your IdentityServer can be done
+Account, and OpenID Connect. In addition, you can find handlers for many
 other authentication providers
 [here](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers).
 
 #### Add Google support
 
+:::note[`Microsoft.AspnetCore.Authentication.Google` no longer maintained]
+Before .NET 10, a package `Microsoft.AspnetCore.Authentication.Google` was provided by Microsoft. Starting with .NET 10,
+Microsoft [stopped shipping new versions of the `Microsoft.AspnetCore.Authentication.Google` package](https://github.com/dotnet/aspnetcore/issues/61817).
+
+Starting with .NET 5, Google started shipping the [`Google.Apis.Auth.AspNetCore3`](https://www.nuget.org/packages/Google.Apis.Auth.AspNetCore3/)
+package. We recommend using this package going forward.
+:::
+
 To use Google for authentication, you need to:
 
-- Add the `Microsoft.AspNetCore.Authentication.Google` NuGet package to
-  the IdentityServer project.
-- Register with Google and set up a client.
-- Store the client id and secret securely with *dotnet user-secrets*.
-- Add the Google authentication handler to the middleware pipeline and configure
-  it.
+- Add the `Google.Apis.Auth.AspNetCore3` NuGet package to the IdentityServer project.
+- Register with Google and [set up a client](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-9.0#create-the-google-oauth-20-client-id-and-secret).
+- Store the client id and secret securely with `dotnet user-secrets`.
+- Add the Google authentication handler to the middleware pipeline and configure it.
 
-See  [Microsoft's
-guide](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-8.0#create-a-google-api-console-project-and-client-id)
+See [Microsoft's guide](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-9.0#create-the-google-oauth-20-client-id-and-secret)
 for details on how to register with Google, create the client, and store the
-secrets in user-secrets. **Stop before adding the authentication middleware and
+secrets in user secrets. **Stop before adding the authentication middleware and
 Google authentication handler to the pipeline.** You will need an
 IdentityServer specific option.
 
-Add the following to `ConfigureServices` in
-`src/IdentityServer/HostingExtensions.cs`:
+Add the following to `ConfigureServices` in `src/IdentityServer/HostingExtensions.cs`:
 
 ```cs
 // Program.cs
 builder.Services.AddAuthentication()
-    .AddGoogle("Google", options =>
-    {
-        options.SignInScheme = IdentityServerConstants.ExternalCookieAuthenticationScheme;
-
-        options.ClientId = builder.Configuration["Authentication:Google:ClientId"];
-        options.ClientSecret = builder.Configuration["Authentication:Google:ClientSecret"];
-    });
+    .AddGoogleOpenIdConnect(
+        authenticationScheme: GoogleOpenIdConnectDefaults.AuthenticationScheme,
```

Comment:
I think you don't need to set authenticatScheme - just displayName should be enough to change the way the button is rendered.

Reply:
Needed unfortunately because of their funky method signature.